SUPPORT.md

```diff
@@ -0,0 +1,52 @@
+# Support and Compatibility for fabric-chaincode-node
+
+Github is used for code base management, issues should reported in the [FABCN](https://jira.hyperledger.org/projects/FABCN/issues/) component in JIRA.
+
+
+## Summary of Compatibility
+
+Summary of the Fabric versions and the default node runtimes they use to host chaincode and contracts; along side the version of the node modules and which runtimes they are compatible with.
+
+|                       | Fabric 1.4 (node v8) | Fabric v2.0 (node v12) |
+| --------------------- | -------------------- | ---------------------- |
+| node modules **v1.4** |         Yes             |      Yes                  |
+| node modules **v2.0** |         No             |          Yes              |
+
+This is the case by default, however the docker image used to host the chaincode and contracts can be altered by configuring the Fabric Peer.  Set the environment variable `CORE_CHAINCODE_NODE_RUNTIME` on the peer to the name of the docker image.
+
+## Compatibility
+
+The key elements are : 
+
+- the version of the Fabric Contract npm modules used
+- the version of the node.js runtime used to run the code
+- When starting a chaincode container to run a Smart Contract the version of the runtime that is used is determined by these factors:
+
+Fabric v1.4.2, and Fabric v2.0.0 will by default start up a docker image, that is defined by the version of Fabric in use.
+
+
+
+With Fabric v2.0.0, the chaincode container can be configured to be started by other means, and not the Peer. In this case, the environment used is not in the control of Fabric.
+ 
+
+npm modules that are produced are the `fabric-contract-api`, `fabric-shim`, `fabric-shim-crypto` & `fabric-shim-api`
```

Comment:
Remove the the?

Reply:
and I was looking for ages for the double 'the'... must be missing it..

Reply:
Think it should be worded "The npm module that are produced are" on second thoughts so

move the the